python/interpret-core/interpret/glassbox/_ebm/_ebm.py

```diff
@@ -219,6 +248,7 @@ def visualize(self, key=None):
             )
 
             return figure
+        # FIXME
```

Comment:
If `explanation_type` is neither `"global"` nor `"local"` we end the function without returning. It is bad practice to return only optionally. Should maybe `None` be returned or a `NotImplementedError` or `RuntimeError` be raised?

Reply:
I agree.  It should probably raise a ValueError.